example/example-server.js

```diff
@@ -0,0 +1,85 @@
+import express from 'express';
+import {service, config} from '../src/index';
+
+const name = 'test-server';
+const port = 10002;
+
+const app = express();
+let server = null;
+
+app.get('/_health', (req, res) => {
+	res.sendStatus(200);
+});
+
+const instance = {};
+
+const init = async (configuration) => {
+	const {identifier, deregister} = await service(configuration);
+
+	Object.assign(instance, {identifier, deregister});
```

Comment:
why not do `const instance = Object.assign({}, {identifier, deregister});`. That was the instance it local to the method